### ISSUE TITLE
InputNumber: doesn't open numeric keyboard on mobile devices

### ISSUE THREAD

**Issue by alexterenya** [primefaces/primevue#3851]
### Describe the bug

When using InputNumber on mobile devices, it shows regular alphabetical keyboard. Since number input allows input only numbers, it should show numeric keyboard only.

**! Important:** I was suggested in my previous bug report: https://github.com/primefaces/primevue/issues/3803 to use :inputProps="{type: 'number'}", but in this case browser shows up/down arrows, which is not appropriate in my case.
<img width="154" alt="image" src="https://user-images.githubusercontent.com/41677103/231125681-e4eacf40-9e89-4b76-a25f-ad85a34d3144.png">.

I would expect to see a regular numeric input without additional controls, but which opens numeric keyboard on mobile browsers.

### Reproducer

https://primevue.org/inputnumber/

### PrimeVue version

3.26.1

### Vue version

3.x

### Language

ALL

### Build / Runtime

Vue CLI App

**Comment by bunomonteiro** [comment]
For those experiencing issues with mobile keyboard display, use the "inputmode" property. This property can be configured in two ways:

1- If the component API has the "inputProps" property (such as the InputNumber component):
```html
<InputNumber :inputProps="{ inputmode: 'numeric' }" />
```

2- If the component is the input itself (as in InputMask):
```html
<InputMask inputmode="numeric" />
```

The "inputmode" supports the following values: none, text, decimal, numeric, tel, search, email, and url.

**Comment by matiasnl23** [comment]
@bunomonteiro Hi, does that work for you? I tried the same, and indeed, it shows the number keypad on Android. However, when I type the number, it does nothing. The input only reflects the change when I press semicolon. Additionally, the backspace does not seem to work.

**Comment by Mecowhy37** [comment]
Same here, exactly as @bunomonteiro described.

![434068182_757473423027767_390356083881119772_n](https://github.com/primefaces/primevue/assets/70144752/c00b5f84-3f66-474a-aeb4-ab3d6d54c753)

![image](https://github.com/primefaces/primevue/assets/70144752/2112a6e9-7f28-430a-92fc-e796cbeb830c)

**Comment by bunomonteiro** [comment]
Yeah, the InputNumber component is being somehow obstructed from receiving the input event. The keyboard is displayed, but the inputs are not being forwarded to the component.

I've created a reproduction example of the issue: [snippet](https://stackblitz.com/edit/6m68gb?file=src%2FApp.vue)

**Comment by MattAlign** [comment]
iPhone seems to work fine with InputNumber component. But when i try to use it on an android i can't delete the numbers and have similar behaviours as described above
.
I also have a passcode on my app which doesn't allow me to add in any numbers on android but seems to work fine on iPhone.

all the input fields seem to work fine on Desktop
